tools/setup_xcode_sdks.py

```diff
@@ -0,0 +1,200 @@
+#!/bin/python
+
+# Update these lists if you need different SDK versions!
+
+iphoneos_versions = ["10.0", "9.3", "9.2", "8.2"]
```

Comment:
@montegoulding IMHO it's important that this script fails if it can't find all the SDKs needed for a successful build.  I think this script is the only place where that info is stored (I think the iOS SDK versions could maybe be parsed from the top-level Makefile, but I'm pretty certain that the MacOS SDK versions aren't available anywhere but here).

Do we need to deal with this as part of this PR?